plugins/modules/ilo_redfish_info.py

```diff
@@ -37,79 +38,158 @@
       - Base URI of iLO.
     type: str
   username:
+    required: false
     description:
       - Username for authenticating to iLO.
     type: str
   password:
+    required: false
     description:
       - Password for authenticating to iLO.
     type: str
   auth_token:
+    required: false
     description:
       - Security token for authenticating to iLO.
     type: str
+    version_added: 2.3.0
   timeout:
+    required: false
     description:
-      - Timeout in seconds for HTTP requests to iLO.
-    default: 10
+      - Timeout in seconds for URL requests to iLO.
+      - The default value for this param is 10 but that is being deprecated
+        and it will be replaced with 60 in community.general 5.7.0.
     type: int
+  cert_file:
+    required: false
+    description:
+      - absolute path to the server cert file
+    type: str
+  key_file:
+    required: false
+    description:
+      - absolute path to the server key file
+    type: str
 author:
-    - "Bhavya B (@bhavya06)"
+  - Bhavya B (@bhavya06)
+  - Gayathiri Devi Ramasamy (@Gayathirideviramasamy)
+  - T S Kushal (@TSKushal)
+  - Varni H P (@varini-hp)
+  - Prativa Nayak (@prativa-n)
 '''
 
 EXAMPLES = '''
   - name: Get iLO Sessions
     community.general.ilo_redfish_info:
       category: Sessions
       command: GetiLOSessions
-      baseuri: "{{ baseuri }}"
-      username: "{{ username }}"
-      password: "{{ password }}"
-    register: result_sessions
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get physical drive details
+    ilo_redfish_info:
+      category: Systems
+      command: GetPhysicalDrives
+      baseuri: "***.***.***.***"
+      username: "abcxyz"
+      password: "******"
+
+  - name: Get logical drive details
+    ilo_redfish_info:
+      category: Systems
+      command: GetLogicalDrives
```

Comment:
Isn't this the same as "GetVolumeInventory" (meaning it steps through Volume resources to get the logical drive info?)

Reply:
Similar to the above comment. SmartStorage is accessed here.

Reply:
Can we change this to include the term "SmartStorage" in the command name to align with your HPE-specific naming?

Reply:
Since both the name are different i.e. GetVolumeInventory and GetLogicalDrives, do we need to specify smart storage.

Reply:
Changing **"GetLogicalDrives"** to **"GetSmartStorageLogicalDrives"**

Reply:
Resolving this conversation, thanks!